content/Solutions/Integrations/SMBClustering.md

```diff
@@ -101,30 +120,64 @@ Click **CREATE SHARE**.
 1. Enter a unique **Name** for the share.
 2. Choose an **ACL** to apply to the share.
    {{< expand "Current Options (click to expand)" "v" >}}
-   * **POSIX_OPEN** - Template that grants read, write, and execute permissions to all users.
-   * **POSIX_RESTRICTED** - Template that grants read, write, and execute to owner and group, but not other. The template may optionally include the special-purpose 'builtin_users' and 'builtin_administrators' groups as well as Domain Users and Domain Admins groups in Active Directory environments.
+   * **POSIX_OPEN** - Grants read, write, and execute permissions for all users.
+   * **POSIX_RESTRICTED** - Grants read, write, and execute to owner and group, but not others. The template may optionally include the special-purpose 'builtin_users' and 'builtin_administrators' groups and Domain Users and Domain Admins groups in Active Directory environments.
```

Comment:
might or could instead of may, also why are Domain Users and Domain Admins init-capped?
```suggestion
   * **POSIX_RESTRICTED** - Grants read, write, and execute to owner and group, but not others. The template might optionally include the special-purpose 'builtin_users' and 'builtin_administrators' groups and Domain Users and Domain Admins groups in Active Directory environments.
```